content/200-concepts/050-overview/100-what-is-prisma/index.mdx

```diff
@@ -266,17 +266,17 @@ Introspection and Prisma Migrate do not yet support the MongoDB connector. To us
 
 With **Prisma Migrate**, Prisma's integrated database migration tool, the workflow looks as follows:
 
-1. Manually adjust your [Prisma data model](../../components/prisma-schema/data-model)
+1. Manually adjust your [Prisma data model](/concepts/components/prisma-schema/data-model)
 1. Migrate your development database using the `prisma migrate dev` CLI command
 1. Use Prisma Client in your application code to access your database
 
-![Typical workflow with Prisma Migrate](../../../doc-images/prisma-migrate-development-workflow.png)
+![Typical workflow with Prisma Migrate](/doc-images/prisma-migrate-development-workflow.png)
```

Comment:
Do these really work this way?

Reply:
Locally they work, yes. And the preview shows them so im happy they should work when deployed. Although moving forward, these should also be moved to our cloudinary account

Reply:
Unfortunately not all do work for me, for example: https://deploy-preview-2140--prisma2-docs.netlify.app/docs/guides/general-guides/database-workflows